src/pages/join_info.tsx

```diff
@@ -1,18 +1,50 @@
 import React, { useState } from "react"; 
 import type { NextPage } from "next";
 import styled from "@emotion/styled"
+import Router from "next/router";
+import theme from "../styles/theme";
 
 const Join: NextPage = () => {
+  const [userNickname, setUserNickname] = useState<string>("");
+  const [checkButton, setCheckButton] = useState<boolean>(false); 
+
+  const handleNickname = (e: React.ChangeEvent<HTMLInputElement>) => {
+    setUserNickname(e.currentTarget.value);  
+  }
+
+  const handleButton = (e: React.MouseEvent<HTMLButtonElement>) => {
```

Comment:
p5: 이렇게도 네이밍 해봐도 좋을 거 같아서 의견남겨 보왔습니다. 
```suggestion
  const onHandleButton = (e: React.MouseEvent<HTMLButtonElement>) => {
```

Reply:
=> handleOnClick